src/broadword.rs

```diff
@@ -5,11 +5,13 @@
 #[cfg(feature = "intrinsics")]
 use crate::intrinsics;
 
-pub(crate) const ONES_STEP_4: usize = 0x1111111111111111;
+pub(crate) const ONES_STEP_4: u64 = 0x1111111111111111;
 pub(crate) const ONES_STEP_8: usize = 0x0101010101010101;
+pub(crate) const ONES_STEP_8_U64: u64 = 0x0101010101010101;
```

Comment:
```suggestion
pub(crate) const ONES_STEP_8: u64 = 0x0101010101010101;
```
Could you unify them into ONES_STEP_8 and cast it if necessary? This is because:
- It's redundant to have similar constants.
- The type difference where ONES_STEP_4 is u64 and ONES_STEP_8 is usize makes the code harder to understand.